include/ajax.tasks.php

```diff
@@ -336,20 +336,18 @@ function massProcess($action, $w=null)  {
                 $info[':title'] = sprintf('Reopen %s',
                          _N('selected task', 'selected tasks', $count));
```

Comment:
@Chefkeks I finally see what you mean. Considering that we already have precedence in the ticket status dialog, I think it makes sense to keep things consistent too. The main problem is that I didn't contribute much to the tasks feature, so it's difficult for me to get the context of what's written there. Please feel free to make the adjustments you feel necessary.